### DISCUSSION TITLE
V-model and v-for

### DISCUSSION THREAD

**Discussion by thomastiko** [quasarframework/quasar#15406] [General - Components / Directives / etc] [answered]
Hi!
What value do I have to give to the v-model of the carousel so that the first slide is not blank?

Thanks for your help! :)

**Carousel:**
<img width="578" alt="image" src="https://user-images.githubusercontent.com/92720511/217530091-351d067a-efff-43f3-852d-be42aeba841c.png">

**data:**
      courses: [
        {
          arr: [
            {
              id: 1,
              title: "Betriebswirtschaftslehre",
              lv: "Einführung in die Bwl",
              lvType: "LVP",
              ects: 4,
              sst: 2,
              grade: null,
              completed: false,
              inProgress: false,
            },
            {
              id: 2,
              title: "Wirtschaftsrecht",
              lv: "Einführung in die Bwl",
              lvType: "LVP",
              ects: 4,
              sst: 2,
              grade: null,
              completed: false,
              inProgress: false,
            },
            {
              id: 3,
              title: "Mathematik",
              lv: "Einführung in die Bwl",
              lvType: "LVP",
              ects: 4,
              sst: 2,
              grade: null,
              completed: false,
              inProgress: false,
            },
          ]
        },
        {
          arr: [
            {
              id: 4,
              title: "Sozioökonomie",
              lv: "Einführung in die Bwl",
              lvType: "LVP",
              ects: 4,
              sst: 2,
              grade: null,
              completed: false,
              inProgress: false,
            },
            {
              id: 5,
              title: "Wirtschaftskommunikation",
              lv: "Einführung in die Bwl",
              lvType: "LVP",
              ects: 4,
              sst: 2,
              grade: null,
              completed: false,
              inProgress: false,
            },
            /*{
              id: 6,
              title: "Volkswirtschaftslehre",
              lv: "Einführung in die Bwl",
              lvType: "LVP",
              ects: 4,
              sst: 2,
              grade: null,
              completed: false,
              inProgress: false, 
            },*/
          ]
        }
      ]

**Comment by MilosPaunovic** [comment]
You need to give it a value of the name of one of your `q-carousel-slide` items.
If you need more help on this topic, please provide a Codepen reproduction for someone to be able to play with it.

**Reply by thomastiko** [reply]
I have tried every possible name but nothing works. When I enter courses[0].arr[0].title the first page is not blank but you can't switch between the slides anymore.
Hier is a Codepen with my work but more simply presented. 
https://codepen.io/thomastiko/pen/WNKBdRz
Thank you very much for your help!

**Reply by MilosPaunovic** [reply]
Is there a reason why the array of options isn't flattened to a same level?

**Reply by thomastiko** [reply]
I have chosen the division this way because:
I would like to show 3 items per slide. So a slide should always be filled with up to 3 items and only when it is full, a new q-carousel-slide should be created. If this is also possible with another array structure, you can choose that too. That was the only structure I could think of to make it work.

**Comment by MilosPaunovic** [comment, marked as answer]
I guess you're looking for something like this. It could, at least, point you in the right direction.
https://codepen.io/milospaunovic/pen/abjrKLJ?editors=1011

**Reply by thomastiko** [reply]
It worked!! Thank you so much :)
I owe you a beer man 🍺 😄
